apimachinery/src/meta/intstr.rs

```diff
@@ -45,11 +46,11 @@ impl PartialEq<Integer> for IntOrString {
     }
 }
 
-impl PartialEq<AsRef<str>> for IntOrString {
-    fn eq(&self, other: &AsRef<str>) -> bool {
+impl PartialEq<str> for IntOrString {
```

Comment:
Fwiw, the 2018 syntax (`dyn AsRef`) highlighted that I didn't actually want the original version of this - and Rust won't let me do a blanket `impl<T: AsRef<str>> PartialEq<T> for IntOrString`